### DISCUSSION TITLE
classes/class_editorsceneformatimporterblend

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#158] [User-contributed notes]
# classes/class_editorsceneformatimporterblend

Inherits: EditorSceneFormatImporter< RefCounted< Object Importer for Blender's.blend scene file format. Description: Imports Blender scenes in the.blend file format through the glTF 2.0 3D import p...

https://docs.godotengine.org/en/stable/classes/class_editorsceneformatimporterblend.html

**Comment by emanvidmaker** [comment]
# ProTip:

Note that when it says:
> Internally, the EditorSceneFormatImporterBlend uses the Blender glTF "Use Original" mode to reference external textures.

That means that it will look up where the textures at and reference it; If the texture file is outside of the project folder it struggles to find it and import it.

I recommend that all the textures referenced in the blend file are inside Godot Project file to avoid Import issues.
- this applies to Godot 4.3, as of writing this
